drivers/SmartThings/zwave-switch/src/multi-metering-switch/multi_metering_switch_configurations.lua

```diff
@@ -17,110 +17,110 @@ local devices = {
     MATCHING_MATRIX = {
       mfrs = 0x0086,
       product_types = 0x0003,
-      product_ids = 0x0084,
-      children = 1
+      product_ids = 0x0084
     },
     CONFIGURATION = {
-      child_switch_device_profile = "metering-switch"
+      child_switch_device_profile = "metering-switch",
+      children = 1
```

Comment:
this should be unset for any device that's not one of the two zooz devices in the DTH

Reply:
@greens I deleted this file due to needless use. Now the logic is as in DTH. The only difference is the lack of one fingerprint (Zooz Double Plug) beacuse of moving it to another driver (zwave-dual-switch).